pylemonway/pylemonway.py

```diff
@@ -24,6 +24,16 @@ def __repr__(self):
     def is_blocked(self):
         return self.BLOCKED or False
 
+    def get_id(self):
+        return self.ID
+
+    def get_lwid(self):
+        return self.LWID
+
+    def serialize(self):
+        import json
```

Comment:
just to embed my new code in a single zone, you can put import at the top of the file now you have merged ;)